cookbook/deploy-contract.mdx

```diff
@@ -11,5 +11,8 @@ To deploy an instance of a compiled smart contract that has already been install
 stellar contract deploy \
     --source S... \
     --network testnet \
-    --wasm-hash <hex-encoded-wasm-hash>
+    --wasm-hash <hex-encoded-wasm-hash> \
+    --alias <your-alias>
 ```
+
+Optionally, you can assign an alias by replacing <your-alias> with your desired alias name for the contract.
```

Comment:
Suggest adding a little more commentary:
````suggestion
    --alias <alias>
```

:::tip
Optionally assign an alias by replacing <your-alias> with your desired alias name for the contract. The alias is a locally stored mapping to the contract address, and can be used in other stellar-cli commands in place of the address.
:::
````